privacyidea/api/validate.py

```diff
@@ -426,44 +356,61 @@ def samlcheck():
     mapping.
     """
     user = request.User
+    result = success = False
```

Comment:
We should differentiate here, the success is a bool but result can also be a dict. The usage is a little confusing in this part.

Reply:
Hm, it looks as if the variables are set later anyways, always.
So what about simply removing them at this point?

Reply:
I remove this in https://github.com/privacyidea/privacyidea/pull/2277/commits/d3cf61aded400cdf5c228b9cd963af4b1f57544d